Modules/ThirdParty/MINC/src/libminc/CMakeLists.txt

```diff
@@ -49,7 +49,9 @@ IF(NOT LIBMINC_EXTERNALLY_CONFIGURED)
 
 
   OPTION(LIBMINC_MINC1_SUPPORT           "Support minc1 file format, requires NETCDF" OFF)
+  OPTION(LIBMINC_BUILD_EZMINC            "Build C++ EMINC library based on libminc1" ON)
```

Comment:
The addition of this with default value ON may need to be switched off to keep this build the same after the update, since ITK isn't using the EZMinc layer.